R/util-cli_logger_wrappers.R

```diff
@@ -0,0 +1,55 @@
+#' cli style console appender for gsm
+#'
+#' @param level warning level that maps to log4r
+#' @param ... should contain message and cli_detail
+#'
+#' @export
+cli_fmt <- function(level, ...) {
```

Comment:
@dpastoor After reading and understanding what you demonstrated, I started working on the rough draft of the "custom" console_appender layout function that mimics gsm's current cli messaging suite. Swapping over to default_log_layout(), simple_log_layout(), bare_log_layout(), logfmt_log_layout(), json_log_layout(), seems to work quite smoothly already using SetLogger() with the appropriate logger object.